step/errors.go

```diff
@@ -8,3 +17,107 @@ type XCPrettyInstallError struct {
 func (e XCPrettyInstallError) Error() string {
 	return e.err.Error()
 }
+
+type NSError struct {
+	Description string
+	Suggestion  string
+}
+
+func NewNSError(str string) *NSError {
+	nserrorPattern := `Error Domain=.* Code=.*UserInfo=.*`
```

Comment:
Is there a reason that you did not include a space between `Code=.*` and `UserInfo=.*` but you did between the `Domain=.*` and `Code=.*` parts?

Reply:
Made the NSError line detection more robust in https://github.com/bitrise-steplib/steps-xcode-archive/pull/309/commits/268115e90df13711a23a58a7125d82b320825f7b.